controllers/ansibleee_controller.go

```diff
@@ -72,6 +72,47 @@ func (r *AnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	//configMapVars := make(map[string]env.Setter)
 	//r.configMapForAnsibleEE(ctx, instance, helper, &configMapVars)
 
+	if instance.Spec.EnablePlugin == true {
+		if len(instance.Spec.Inventory > 0) {
+			fmt.Println("Using dynamic inventory, setting existing inventory to nil")
```

Comment:
Take in account that the jobs are being instantiated in a per-playbook base, so no need to remove the existing inventory from here. It is a matter of the user to choose if for that execution he wants to use a plugin inventory or an inventory file.

So, no need for this check, let ansible handle it